### DISCUSSION TITLE
One-to-one in LinkToAnotherRecord

### DISCUSSION THREAD

**Discussion by nonacc** [nocodb/nocodb#7847] [Features]
**nocodb is the best.** 

and it's potential is enormous.

1 bit is missing for it to be industry leader.

one-to-one relation. it's native, essential and fundamental in any database.

could you prioritize this feature please?

![image](https://github.com/nocodb/nocodb/assets/12987330/2ef6ee6d-2f34-4636-89a2-9f905bf20125)

its gonna be a no brainer to migrate everything to it.

it's gonna drive thousands of companies using it.

nocodb is going to explode with one-to-one relation in place.

if there's any way to help or speed things up, i'm down :)

**Comment by o1lab** [comment]
FYI, [one to one PR is merged ](https://github.com/nocodb/nocodb/pull/7915)

<img width="376" alt="image" src="https://github.com/nocodb/nocodb/assets/5435402/f25a3073-d131-4fb0-b83f-af49379da938">

so coming soon 🚀 ;)

**Reply by nonacc** [reply]
**nocodb-first** is my new default now
i praise its unmatched flexibility :)
